operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/BaseConfigurationService.java

```diff
@@ -0,0 +1,28 @@
+package io.javaoperatorsdk.operator.api.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BaseConfigurationService extends AbstractConfigurationService {
```

Comment:
I don't get this class. Why are we creating loggers this way?

Reply:
This class is just used to be able to have a non-abstract implementation that can be used in the junit extension. I'm debating whether to just put the `DefaultConfigurationService` into core itself and get rid of this class completely but I'm not too sure of the impact on the quarkus extension…

Reply:
oki, 
(will take a look on the quarkus side to understand these aspects better)